NuRadioReco/detector/ARIANNA/arianna_detector_200.json

```diff
@@ -0,0 +1,279 @@
+            "station_id": 101
+        },
+        "2": {
+            "adc_id": null,
+            "adc_n_samples": 256,
+            "adc_nbits": null,
+            "adc_sampling_frequency": 1.0,
+            "adc_time_delay": null,
+            "amp_reference_measurement": "300-03-01",
+            "amp_type": "300",
+            "ant_comment": "close to center, pointing downwards (2/4)",
+            "ant_deployment_time": "{TinyDate}:2018-12-05T00:00:00",
+            "ant_orientation_phi": 0.0,
+            "ant_orientation_theta": 180.0,
+            "ant_position_x": -2.1,
+            "ant_position_y": 2.1,
+            "ant_position_z": -3,
+            "ant_rotation_phi": -45.0,
+            "ant_rotation_theta": 90.0,
+            "ant_type": "createLPDA_100MHz",
+            "cab_id": "17-11",
+            "cab_length": 5.0,
+            "cab_reference_measurement": null,
+            "cab_time_delay": 19.8,
+            "cab_type": "LMR_400",
+            "channel_id": 1,
+            "commission_time": "{TinyDate}:2018-12-05T00:00:00",
+            "decommission_time": "{TinyDate}:2038-01-01T00:00:00",
+            "station_id": 101
+        },
+        "3": {
+            "adc_id": null,
+            "adc_n_samples": 256,
+            "adc_nbits": null,
+            "adc_sampling_frequency": 1.0,
+            "adc_time_delay": null,
+            "amp_reference_measurement": "300-03-02",
+            "amp_type": "300",
+            "ant_comment": "close to center, pointing downwards (3/4)",
+            "ant_deployment_time": "{TinyDate}:2018-12-05T00:00:00",
+            "ant_orientation_phi": 0.0,
+            "ant_orientation_theta": 180.0,
+            "ant_position_x": -2.1,
+            "ant_position_y": -2.1,
+            "ant_position_z": -3,
+            "ant_rotation_phi": 45.0,
+            "ant_rotation_theta": 90.0,
+            "ant_type": "createLPDA_100MHz",
+            "cab_id": "17-10",
+            "cab_length": 5.0,
+            "cab_reference_measurement": null,
+            "cab_time_delay": 19.8,
+            "cab_type": "LMR_400",
+            "channel_id": 2,
+            "commission_time": "{TinyDate}:2018-12-05T00:00:00",
+            "decommission_time": "{TinyDate}:2038-01-01T00:00:00",
+            "station_id": 101
+        },
+        "4": {
+            "adc_id": null,
+            "adc_n_samples": 256,
+            "adc_nbits": null,
+            "adc_sampling_frequency": 1.0,
+            "adc_time_delay": null,
+            "amp_reference_measurement": "300-03-03",
+            "amp_type": "300",
+            "ant_comment": "close to center, pointing downwards (4/4)",
+            "ant_deployment_time": "{TinyDate}:2018-12-05T00:00:00",
+            "ant_orientation_phi": 0.0,
+            "ant_orientation_theta": 180.0,
+            "ant_position_x": 2.1,
+            "ant_position_y": -2.1,
+            "ant_position_z": -3,
+            "ant_rotation_phi": -45.0,
```

Comment:
the antennas should point inwards, we define the rotation of the lpdas by "where the connector points to" So it needs to be -45+180

Reply:
same for the first channel

Reply:
and all upward LPDAs

Reply:
Question: At the moment all antennas pointing to the centre (see detector visualization)
![newplot](https://user-images.githubusercontent.com/56539848/69332262-9a14da00-0c56-11ea-88d4-d3495e62e4fa.png)

How should i change them?

Reply:
hmm that's weird. Maybe something in the visualization script is wrong. I'll have a look later.

Reply:
ok I finally figured out what was wrong. The zenith angle was not correctly defined. In our definition, the zenith angle is only allowed within [0deg, 180deg]. 0deg is pointing upwards (to the zenith), 180deg is pointing downwards. Having a negative zenith angle resulted in undefined behavior.